tests/data/test_minute_bars.py

```diff
@@ -142,6 +142,43 @@ def test_write_one_ohlcv(self):
 
         self.assertEquals(50.0, volume_price)
 
+    def test_precision_after_scaling(self):
+        '''For numbers that don't have an exact float representation,
+        assert that scaling the value does not cause a loss in precision.
+        '''
+        minute = self.market_opens[self.test_calendar_start]
+        sid = 1
+        data = DataFrame(
+            data={
+                'open': [130.23],
+                'high': [130.23],
+                'low': [130.23],
+                'close': [130.23],
+                'volume': [1000]
+            },
+            index=[minute])
+        self.writer.write_sid(sid, data)
+
+        open_price = self.reader.get_value(sid, minute, 'open')
+
```

Comment:
It might be easier to read without these newlines between the read and the assert because then they will fall into little groupings.